### ISSUE TITLE
DestroyUser doesn't actually destroy user

### ISSUE THREAD

**Issue by utdrmac** [Storj/bridge#196]
### Package Versions

```
asura:CodeProjects drmac$ npm -g list storj
/usr/local/lib
└── storj@3.0.4

node --version
v4.4.7
```

### Expected Behavior

Per the title of the function "BridgeClient::destroyUser", one would expect the user to actually be destroyed. This appears to not be the case as after calling destroyUser, confirming delete via email and then attempting to re-register that same email gives the error "Email is already registered"

Attempting to generate a keyPair or attempting to delete (again) the "phantom" user, or really do anything else returns error "User account has not been activated"

Sending a client.resetPassword() request, via un-authenticated client object, to which I do receive the email. Clicking this reset password link, however, does not re-activate the user. The user is still sorta there but unusable.

There is no way to re-activate nor recover this "deleted" user.

### Steps to Reproduce

1. client.createUser()
2. confirm via email link
3. client.destroyUser()
4. confirm delete via email link
5. repeat step 1. receive error message.

**Comment by super3** [comment]
@utdrmac We had this discussion in the office, and I'm trying to remember the our conclusion.

**Comment by utdrmac** [comment]
Well, I only see two options. 1) actually and truly delete the user. from a security standpoint, this is the best option to implement. Give users the ability to remove all traces of themselves from the network and backend databases. If user comes back later, they must re-register.  2) mark the user deleted and give ability to recreate/restore at a later time by resetting password. though, if they've lost access to associated email account, then i guess it's non-recoverable.

**Comment by bookchin** [comment]
This is being included in #222. You will be able to re-activate an deactivated account with:

```
POST /activations
{ "email": "<email>", "redirect": "<optional redirect url for web app>" }
```

This will send an activation email again to reactivate the account.
